### PR TITLE
Add a CONTRIBUTING.md file.

### PR DESCRIPTION
**Pull request by oSoMoN**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,43 +1,48 @@
...
 
 ## Checkout & Run
 
 Grab the source:
-```
+```sh
 git clone https://github.com/canonical/ubuntu-desktop-installer.git
 cd ubuntu-desktop-installer
 git submodule update --init --recursive
+cd -
```
Comment (MarcusTomlinson) [inline README.md:14]: I wouldn't add this personally. All of the 'Checkout & Run' instructions are written such that you can run them from top to bottom. If you `cd -` it'll put you in the wrong directory for the next set of commands.

File: README.md [hunk 2]
```diff
@@ -1,43 +1,48 @@
...
 Install dependencies:
-```
+```sh
 cd subiquity_client
 flutter pub get
 cd subiquity
 make install_deps
 cd ../../ubuntu_desktop_installer
 flutter pub get
+cd -
```
Comment (MarcusTomlinson) [inline README.md:33]: I'd remove this too.

**Review by MarcusTomlinson — CHANGES_REQUESTED** [review]
Looks good! Just a few nitpicks.

File: README.md (hunk 2 above)
Comment (oSoMoN) [reply README.md]: Removed.

File: README.md (hunk 1 above)
Comment (oSoMoN) [reply README.md]: Removed.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
+# Contributing
+
+## Pull requests
+
+Changes to this project should be proposed as pull requests on Github at: <https://github.com/canonical/ubuntu-desktop-installer>.
```
[the remark this answers is not in the document]
Comment (oSoMoN) [reply CONTRIBUTING.md]: Fixed.

**Review by MarcusTomlinson — APPROVED** [review]
Thanks!

**Pull request merged**